HTML Greek symbol edition (#3058)
Codecademy/docs: content/html/concepts/entities/greek_symbol.md

Reviewer: The path to this file should be `content/html/concepts/entities/terms/greek-symbols/greek-symbols.md`

---
Title: "Greek symbols"

> Description: "Greek symbols are frequently employed in scientific, mathematical, and technological applications. It can use as HTML Character Entities"

Reviewer: ```suggestion
Description: "Greek symbols are frequently employed in scientific, mathematical, and technological applications. They can be used via HTML entities."

```

Subjects:
  - "Web Development"
  - "Web Design"
Tags:
  - "Characters"
  - "Symbol"
CatalogContent:
  - "learn-html"
  - "paths/front-end-engineer-career-path"
---

> Greek symbols are frequently employed in scientific, mathematical, and technological applications. It can use as HTML Character Entities and by using Unicode values.

Reviewer: ```suggestion
Greek symbols are frequently employed in scientific, mathematical, and technological applications. They can be used via HTML entities and by using Unicode values.

```

> You can simply include Greek symbols into your HTML projects using either way. Simply select the way that is most suitable for your requirements.

Reviewer: ```suggestion
```
Remove this line, it doesn't add anything.

## Example

> You may use the HTML entity: to display the Greek letter "alpha" ().

Reviewer: ```suggestion
Using the HTML entity to display the Greek letter "alpha":
```

```
&alpha;
```

> You can use the following format to represent Greek symbols "alpha" in Unicode:

Reviewer: ```suggestion
The following format represents the Greek letter "alpha" in Unicode:
```

```
&#x03B1;
```

## Greek capital letter normal

> | Symbol | HTML Code | HTML Entity  | CSS Code | Hex Code  | Unicode | Description                   |

Reviewer: You need to correct the HTML code column.
